### DISCUSSION TITLE
ProteinKit

### DISCUSSION THREAD

**Discussion by philmui** [chdominguez/Atomic#43] [General]
In trying to compile Atomic, this message shows up:

"Atomic/Atomic.xcodeproj Missing package product 'ProteinKit'"

I cannot find an open source project called ProteinKit.  Is there another dependency here that should be linked to Atomic?

**Comment by chdominguez** [comment]
Sorry for the confusion Phil. I made ProteinKit private a while ago because it was very incomplete. I forgot to make it public again. 

Fixed. You can find ProteinKit [here](https://github.com/chdominguez/ProteinKit).
